scripts/upload-housing.js

```diff
@@ -116,18 +124,18 @@ const main = async () => {
   const app = initializeApp();
   const db = getFirestore(app);
 
-  // Check if the collection exists
+  // Check if the spotlight collection exists
   const collections = (await db.listCollections()).map((c) => c.id);
   if (!collections.includes("spotlights")) {
-    warnAndExit(`ERROR! housing must match an existing collection id: ${collections.join(
+    warnAndExit(`ERROR! spotlights must match an existing collection id: ${collections.join(
       ", "
     )}.
      `);
   }
```

Comment:
I feel like this check isn't necessary anymore since the spotlights collection will always exist. There was an earlier comment about checking all the documents inside of the spotlights collection?

Reply:
Yeah, in the other file it checks if the collection already exists, and if it doesn't, you have to use a `newId` flag. I guess I'm not sure of the purpose to that, so I didn't include it here.

Reply:
Ooooh, I think I get the purpose. Figured out how to do it!